constants/index.js

```diff
@@ -2,16 +2,24 @@ const ROLE_ID = {
   USER: 1,
   CREATOR: 2,
   ADMIN: 3,
-  SUPERADMIN: 4
-}
+  SUPERADMIN: 4,
+};
 const ROLE_NAME = {
-  USER: 'USER',
-  CREATOR: 'CREATOR',
-  ADMIN: 'ADMIN',
-  SUPERADMIN: 'SUPERADMIN'
-}
+  USER: "USER",
+  CREATOR: "CREATOR",
+  ADMIN: "ADMIN",
+  SUPERADMIN: "SUPERADMIN",
+};
+
+const RESOURCE = {
+  Post: "Post",
+  Comment: "Comment",
+  User: "User",
+  UserProfile: "UserProfile",
+};
 
```

Comment:
Add constant variable for HTTP METHODS to use them in your middleware
const HTTP_METHOD = {
    DELETE: "DELETE",
    PUT: "PUT",
    GET: "GET",
    POST: "POST"
};